lang/query/src/info/collect.rs

```diff
@@ -338,12 +338,16 @@ impl CollectInfo for DotCall {
 
 impl CollectInfo for Hole {
     fn collect_info(&self, collector: &mut InfoCollector) {
-        let Hole { span, metavar, inferred_type, inferred_ctx } = self;
+        let Hole { span, metavar, inferred_type, inferred_ctx, args } = self;
         if let Some(span) = span {
             let info = HoleInfo {
                 goal: inferred_type.print_to_string(None),
                 metavar: metavar.map(|mv| format!("?{}", mv.id)),
                 ctx: inferred_ctx.clone().map(Into::into),
+                args: args
```

Comment:
I think we don't need to traverse the args, since the do not have location spans.